validator/client/service.go

```diff
@@ -114,7 +114,7 @@ func (v *ValidatorService) Start() {
 		return
 	}
 
-	valDB, err := db.NewKVStore(v.dataDir, pubkeys)
+	valDB, err := db.NewKVStoreWithPublicKeyBuckets(v.dataDir, pubkeys)
```

Comment:
The naming was changed because I created a second function that returns a store without any key buckets. As stated in the comment below, this should be reverted and the old function should be used.